bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,32 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
 #----alsa install
 sudo apt-get install -y alsa-base alsa-utils
```

Comment:
Locale is only to avoid warnings.
IPV6 and extra DNS depends on the internet settings. If IPV4 and IPV6 are both enabled (it is by default) you could have intermittent problems. Sametimes the connection go over IPV4 and sometime using IPV6. It is happening with my rpi. I think only one (IPV4 or IPV6) should be enabled to make sure everything will works. I don't know if a user will detect the intermittent problem and  use the parameter. But I could change that if you prefer.